### ISSUE TITLE
Development Review - 13 Sep 2016

### ISSUE THREAD

**Issue by philcarrillo** [weenhanceit/autism-funding#25]
This is a set of notes made when reviewing the Develop Branch of the site on 13 Sep 2016.
- Need to review the experience of first time users or users who have not defined a province.  The language  for the user home page should differentiate between users with an unsupported province and users who have not defined a province.  It may be confusing to BC residents on initial log in to have a bunch of messages saying "you're not supported" because they've yet to enter their province.
- BC users who have no funded children defined see a rather blank home page with no prompt to enter a funded child.  There should be a message on the home page for this case. See Issue #26.
- In RTP entry screen, modifications of My Profile data seem to be working upon hitting the save button.   But the form was not created (there were missing data elements) and any data that I may have entered would be lost.  Suggest we would need a form "validator" that would check for completeness, but that a user should be able to save a form that is unsuitable for submission.
- When the user does change profile information, they should get some sort of notification that the profile information has been updated.  There may have been inadvertent key strokes and the user may not notice that they have modified some profile data.
- Need to review where users is taken after the save of an RTP form.  I would suggest users home page.  Presently taken to a form summary page that does not have any links on it.
- Review Child in Care of Ministry field.   This is presently editable in the RTP form - but this may be due to laziness on Phil's part in not implementing this field in the profile data.
- on the users home page, where there are multiple funded children, the text surrounding the Invoices and Expenses is repeated.   I really like the text content, but I worry that they take valuable real estate on the page - and their usefulness diminishes as the user becomes more familiar with the site.   Perhaps this text may be factored out and appear once at the bottom of the page, be moved to a tool tip or only appear if not forms have been initiated.

**Comment by lcreid** [comment]
After save of RTP, I sent the user to a page with the form, where I planned to have a "Print" button, on the assumption that that's the most common next step. I can see a lot of variations on what the user might want to do. In the meantime, I think I'll just add "Home" and "Edit" links to the view page for an RTP. I'm open to other ideas for sure.

**Comment by lcreid** [comment]
Child in Care of Ministry field: Yes, it's editable because it isn't in the profile data.

**Comment by lcreid** [comment]
In RTP entry screen:
- Yes, the validations are incorrect, in that we agreed that we would allow the user to save a very incomplete form, and not allow printing until the form was complete enough. I will create a task for this (Issue #27)
- I tried entering an incomplete form with the validations as they were, and I didn't lose the data I had entered. Am I misunderstanding what you're reporting about the way the code works?
